Cargo.lock

```diff
@@ -1712,9 +1668,9 @@ dependencies = [
 
 [[package]]
 name = "datafusion"
-version = "21.0.0"
+version = "21.1.0"
```

Comment:
> fix: panic with datafusion 21

Two questions:
1. What was the problem?
2. How was the problem fixed?

Reply:
just fixed a version conflict https://github.com/apache/arrow-datafusion/compare/21.0.0...21.1.0